lib/chore/unit_of_work.rb

```diff
@@ -19,7 +20,7 @@ def initialize(*) #:nodoc:
 
     # The current attempt number for the worker processing this message.
     def current_attempt
-      previous_attempts + 1
+      previous_attempts.to_i + 1
```

Comment:
I came across this and it seemed like a subtle bug. Can anyone think of a reason _not_ to do this?

Reply:
Under what circumstances is this not an integer?  As far as I can tell, both the filesystem and sqs consumer pass an integer through here.  If the specs are passing a string, then I'd say the specs are wrong.

Reply:
In reality we never pass in anything but an integer as far as I can tell. I noticed this because by _looking_ at `UnitOfWork` it appears that `:previous_attempts` could at the very least be `nil`. But yeah, specs should cover that.

Reply:
I think in that case we should remove this change as I see that as both a Ruby language design problem and a failed contract for the consumer.  I think better documentation could make it clearer what is expected to be nil vs. not nil.

Reply:
_nod_ I will back out the change as part of addressing forthcoming feedback :)